searchresults/index.html

```diff
@@ -4,17 +4,14 @@
   <title>Flight Booking: Search Results</title>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width,initial-scale=1.0,shrink-to-fit=no" />
+  <link rel="stylesheet" href="screen.css" />
   <link rel="stylesheet" href="http://basehold.it/20" />
   <link rel="stylesheet" href="../screen.css" />
 </head>
 <body>
   <main>
     <h1>Search Results</h1>
-    <!-- todo: add form that replicates inputs from
-    the first form and allows editing -->
-
-
-  <!--  <form id="userform" action="#null" method="get">
+    <form id="Fligtupdate" action="#null" method="get">
```

Comment:
this is unimportant but all of your ids should be lowercase so it'd be "flightupdate" not "Flightupdate"